app/assets/javascripts/float-kit/addon/services/toasts.js

```diff
@@ -14,6 +14,8 @@ export default class Toasts extends Service {
    *
    * @param {Object} [options] - options passed to the toast component as `@toast` argument
    * @param {String} [options.duration] - The duration (ms) of the toast, will be closed after this time
+   * @param {Boolean} [options.autoClose=true] - When true, the toast will auto close after the duration
+   * @param {Boolean} [options.forceAutoClose=false] - When true, mouseover will not cancel the auto-close timer
```

Comment:
```suggestion
   * @param {Boolean} [options.forceAutoClose=false] - When true, toast will still autoClose following mouseover event
```
Remove the double-negative "not cancel" ?